.gitignore

```diff
@@ -2,3 +2,6 @@
 .mypy_cache/*
 
 __pycache__
+
+.pytest_cache
+.vscode
```

Comment:
can we just copy-paste the most significant parts from gitignore of qcodes here? and forget about this %^&$% issue?

Reply:
I'm not going to stop you :) It's a good idea